src/reynir_correct/errfinder.py

```diff
@@ -325,7 +325,7 @@ def AðvörunHeldur(self, txt: str, variants: str, node: Node) -> AnnotationDict
         start, end = self.node_span(node)
         return AnnotationDict(
             text="'{0}' er sennilega ofaukið".format(txt),
-            detail="Yfirleitt nægir að nota 'en' í þessu samhengi.",
+            detail="Yfirleitt nægir að nota 'en' í þessu samhengi, 'heldur en' telst frekar óformlegt. Það á þó stundum rétt hjá sér í löngum setningum ef langt er frá upphaflega samanburðinum til seinni samanburðarins til að auka skýrleika textans.",
```

Comment:
Could be made clearer, this is not a case of two comparisons but two objects being compared.

Reply:
Changed to 'samanburðarliður', comparison phrase.